s4e-web/src/app/views/settings/intitution-profile/institution-profile.component.html

```diff
@@ -49,7 +49,9 @@ <h2 id="institution-title">{{ activeInstitution.name }}</h2>
                        [error]="error$ | async"
                        [searchable]="true">
   <ng-container description></ng-container>
-  <ng-container caption i18n>Jednostki podległe</ng-container>
+  <h3 caption-inside i18n>
+    <ng-container>Jednostki podległe</ng-container>
```

Comment:
`<ng-container>Jednostki podległe</ng-container>` can be just simple `Jednostki podległe`, because `<ng-container></ng-container>` will be not visible `DOM element` after compilation (only it's content `Jednostki podległe`)